render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+me=$(basename $0)
+
+function showUsage() {
+	cat<<-EOF
+Usage: $me --installer <msi|inno|nuget>
```

Comment:
Yeah, I waffled on this. I created it, removed it, and finally re-introduced it again. But I see your point, and I think it's probably better to not have this `-i` option in the long run, too, and just use `-o`.